src/stories/components/Review/styles.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Review
+
+.review {
+  padding: $spacing-1 $spacing-4;
+  max-width: rem(600px);
+  font-size: $font-size-sm
+}
+
+.review__recipe {
+  font-size: $font-size-sm;
+  font-family: $header-font-family;
+  font-weight: 700;
+  letter-spacing: rem(1px);
+
+  text-transform: uppercase;
+  // TODO: Change to sand-dark, I saw there was already on created
+  //   so will wait until that PR is merged then use it.
```

Comment:
If this is re: the background color, this is a global style so no need to implement it.